client/src/styles/components/_Status.scss

```diff
@@ -45,7 +45,10 @@
 .floating-error {
   display: flex;
```

Comment:
I originally wanted the floating error to be absolute-positionned (to not have to think where we should place it on each page where we use it).
Do you think it's a good idea (with regards to UX/UI).
Pinging @valentinperignon here

Reply:
I started like this, but finally it's more convenient to show the message where the request failed.
And absolute positioning really complicates the display, so i found that it works well like that 😀

Reply:
And I think the main benefit of the float error is to show an error, without blocking the whole page, and to allow interacting with other elements, so it's okay to specify where the error should be displayed !

Reply:
Ok, good to know! It's good for me then :+1: